Mathlib/MeasureTheory/Integral/Bochner.lean

```diff
@@ -1370,6 +1367,9 @@ theorem norm_integral_le_integral_norm (f : α → G) : ‖∫ a, f a ∂μ‖ 
   · rw [integral_non_aestronglyMeasurable h, norm_zero]
     exact integral_nonneg_of_ae le_ae
 
+lemma abs_integral_le_integral_abs {f : α → ℝ} : |∫ a, f a ∂μ| ≤ ∫ a, |f a| ∂μ :=
```

Comment:
I'd argue `f` should be explicit to match the lemma above.

Reply:
I would rather make `f` implicit in the lemma above